README.md

```diff
@@ -1181,6 +1181,7 @@ Update Time, five active automations, webhooks.
   * [SourceForge](https://sourceforge.net/) — Find, Create, and Publish Open Source software for free
   * [Stormkit](https://stormkit.io/) — Integrate building, deploying, and hosting seamlessly with the git flow of your JAMStack or Node.JS app. 50 GB bandwidth and 10m monthly requests for free, including free SSL.
   * [surge.sh](https://surge.sh/) — Static web publishing for Front-End developers. Unlimited sites with custom domain support
+  * [surgeme.xyz](https://surgeme.xyz/host/) — Static web hosting for developers. Unlimited sites with unlimited bandwith and storage.
```

Comment:
Sorry there is no information here, no indication what this services is, no t'c and c's its just a login box void of details.  It could be a porn site, how would we know?